src/components/InputField.tsx

```diff
@@ -14,7 +15,14 @@ interface InputFieldProps {
 }
 
 const FieldWrapper = styled.div`
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
   margin-bottom: 1.5rem;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
 `;
 
 const addErrorStyling = ({error}: {error?: string}): string | null => {
```

Comment:
These interpolation functions are getting out of hand with typing.
Use this type declaration to help
`type InterpolationFunction<T extends object> = (props: T) => string | null;`
May want to place it in a file such as types.ts which will live in utils folder